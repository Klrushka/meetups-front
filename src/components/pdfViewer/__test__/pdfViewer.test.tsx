import React from 'react';
import rendered, { act } from 'react-test-renderer';
import {Pdf} from '../index'


describe('PDF component', () => {
  let component: rendered.ReactTestRenderer;

  it('PDF component', () => {
    act(() => {
      component = rendered.create(<Pdf />);
    });
  
    expect(component.toJSON).toMatchSnapshot()
  });
  
})
