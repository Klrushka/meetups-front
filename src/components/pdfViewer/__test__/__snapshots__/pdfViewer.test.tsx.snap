// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PDF component PDF component 1`] = `[Function]`;
